src/upload.js

```diff
@@ -152,7 +155,8 @@ export default class Upload extends EventEmitter {
           this.alpha,
           this.beta,
           sessIdA,
-          sessIdB
+          sessIdB,
+          { progressCb }
```

Comment:
There is only one value in options right now. Would it make sense to move some of the other parameters to options?

Reply:
I'm not sure which should be moved into `opts`. I view it as additional params that are not required.

@nullpilot do you have an opinion on this?